_posts/en/newsletters/2023-12-20-newsletter.md

```diff
@@ -0,0 +1,992 @@
+keys. The beta release supports CLN and LDK, layer-1 and layer-2
+validation rules, backup and recovery capabilities, and provides a
+reference implementation.
+
+{:#ln-meeting}
+An LN developer [meeting][jul summit] held in July discussed a variety
+of topics, including reliable transaction confirmation at the base
+layer, [taproot][topic taproot] and [MuSig2][topic musig] channels,
+updated channel announcements, [PTLCs][topic ptlc] and [redundant
+overpayment][topic redundant overpayments], [channel jamming mitigation
+proposals][topic channel jamming attacks], simplified commitments, and
+the specification process.
+
+## August
+
+{:#onion-messages}
+[Support][aug onion] for [onion messages][topic onion messages] was
+added to the LN specification in August.  Onion messages allow sending
+one-way messages across the network. Like payments ([HTLCs][topic
+htlc]), the messages use onion encryption so that each forwarding node
+only knows what peer it received the message from and what peer should
+next receive the message. Message payloads are also encrypted so that
+only the ultimate receiver can read it.  Onion messages use [blinded
+paths][topic rv routing], which were added to the LN specification in
+April, and onion messages are themselves used by the proposed [offers
+protocol][topic offers].
+
+{:#backup-proofs}
+Thomas Voegtlin [proposed][aug fraud] a protocol that would allow
+penalizing providers for offering outdated backup states to users. This
+service involves a simple mechanism where a user, Alice, backs up data
+with a version number and signature to Bob. Bob adds a nonce and commits
+to the complete data with a time-stamped signature. If Bob provides
+outdated data, Alice can generate a fraud proof showing Bob previously
+signed a higher version number.  This mechanism isn't Bitcoin-specific,
+but incorporating certain Bitcoin opcodes could enable its use onchain.
+In a Lightning Network (LN) channel, that would allow Alice to claim all
+channel funds if Bob provided an outdated backup, thus reducing the risk
+of Bob deceiving Alice and stealing her balance.  The proposal sparked
+significant discussion.  Peter Todd pointed out its versatility beyond
+LN and suggested a simpler mechanism without fraud proofs, while Ghost43
+highlighted the importance of such proofs when dealing with anonymous
+peers.
+
+{:#tapchan}
+LND added [experimental support][aug lnd taproot] for "simple taproot
+channels", allowing LN funding and commitment transactions to use
+[P2TR][topic taproot] with support for [MuSig2][topic musig]-style
+[scriptless multisignature signing][topic multisignature] when both
+parties are cooperating.  This reduces transaction weight and
+improves privacy when channels are closed cooperatively. LND continues
+to exclusively use [HTLCs][topic htlc], allowing payments starting in a
+taproot channel to continue to be forwarded through other LN nodes that
+don’t support taproot channels.
+
```

Comment:
There are 2 related-in-my-mind items that could fall under a new August heading around "policy":

- January's Continued RBF discussion https://bitcoinops.org/en/newsletters/2023/01/04/#continued-rbf-discussion
- Aug's Proposed changes to Bitcoin Core default relay policy  https://bitcoinops.org/en/newsletters/2023/08/09/#proposed-changes-to-bitcoin-core-default-relay-policy 

a 3th, weaker related item:
- July's Preventing coinjoin pinning https://bitcoinops.org/en/newsletters/2023/06/28/#preventing-coinjoin-pinning-with-v3-transaction-relay

This could be another opportunity to highlight the waiting for confirmation series (even though we do already call it out) which wrapped up end of July.

Just a thread I noticed in my notes of things not covered in this YiR while going through the 2023 newsletters.

Reply:
We already cover the Aug proposed relay changes item, although we associate it with its op_return side in the ordinals paragraph.

For the RBF discussion, we had a whole extended section on that in last year's YIR, and the january item was really just the tail end of that (IMO), so I think it's ok that we skip it.

The coinjoin/v3 item doesn't feel strong enough to me to warrant a parargaph of its own, or even a paragraph with just it and the mempool series (since we already give that a short paragraph).  Hopefully we'll have a lot of v3 stuff next year.

Thanks for going through each of the newsletters again!  I meant to do that myself but got lazy.